app/services/billable_metrics/prorated_aggregations/sum_service.rb

```diff
@@ -32,6 +32,18 @@ def aggregate(options: {})
         result.service_failure!(code: 'aggregation_failure', message: e.message)
       end
 
+      def compute_per_event_prorated_aggregation
+        persisted_events = persisted_query
+          .pluck(Arel.sql("(COALESCE((#{sanitized_field_name})::numeric, 0)) * (#{persisted_pro_rata})::numeric"))
+
+          # NOTE: Added during the period
```

Comment:
Layout/CommentIndentation: Incorrect indentation detected (column 10 instead of 8).